pkg/containerd/events.go

```diff
@@ -206,6 +235,91 @@ func (e *EventMonitor) Close() error {
 	return finalErr
 }
 
+// When namespaced network is enable, if portbinding is bound to 127.0.0.1 we update the DNAT
+// in main CNI DNAT chain (CNI-HOSTPORT-DNAT). The existing DNAT rule are as follows:
+// DNAT       tcp  --  anywhere             localhost            tcp dpt:9119 to:10.4.0.22:80.
+// While, this rule only allows the traffic to get routed to the localhost only we update
+// the rule to allow the traffic to any destination IP address.
```

Comment:
```suggestion
// While this rule only allows the traffic to get routed to the localhost only we update
// the rule to allow the traffic to any destination IP address.
```
Actually, maybe rephrase it more…
```suggestion
// We modify the rule to instead capture all traffic to that port regardless of the
// destination address:
```

This seems to lead to issues that I'm not sure how to resolve; imagine a user has two services, listening on `127.0.0.1:9999` and `0.0.0.0:9999`. This change means we can only have _one_ service instead of two…

Reply:
Users won't be able to re-use `9999` port twice the host-switch proxy will not allow them and will get an error.